dist/images/ovnkube.sh

```diff
@@ -2377,6 +2423,18 @@ ovn-node() {
   fi
   echo "ovnkube_node_certs_flags=${ovnkube_node_certs_flags}"
 
+  ovn_transit_switch_subnet_ipv4_opt=
+  if [[ -n ${ovn_transit_switch_subnet_ipv4} ]]; then
+      ovn_transit_switch_subnet_ipv4_opt="--cluster-manager-v4-transit-switch-subnet=${ovn_transit_switch_subnet_ipv4}"
```

Comment:
hmm wasn't this just `--v4-transit-switch-subnet` ? where did the `--cluster-manager` come from? did that already exist from numan's PR?
EDIT: Ah right it did: https://github.com/ovn-org/ovn-kubernetes/commit/ae27492e2ecdbb6139c2deba9919ca9da51d32cc#diff-25b766c2e6c28d5bead46916736a69559de3aef474c269761b51dde666898b8dR1370